<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimerX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1100</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="Interface">
      <item>
       <layout class="QHBoxLayout" name="Buttons">
        <item>
         <widget class="QPushButton" name="continueTimerButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>36</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>–î–∞–ª—ñ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setTimerButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>36</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>–î–æ–¥–∞—Ç–∏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseTimerButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>36</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>–ü–∞—É–∑–∞</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Timers">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="alarms">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Montserrat</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="text">
               <string>–ë—É–¥–∏–ª—å–Ω–∏–∫–∏:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="nearestTimerText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Montserrat</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="text">
               <string>–ù–∞–π–±–ª–∏–∂—á–∏–π —Ç–∞–π–º–µ—Ä:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="timers">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Montserrat</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="text">
               <string>–¢–∞–π–º–µ—Ä–∏:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="Alarms" stretch="0">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QListWidget" name="alarmsList">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>JetBrains Mono NL</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="autoScrollMargin">
                 <number>16</number>
                </property>
                <property name="tabKeyNavigation">
                 <bool>true</bool>
                </property>
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="dragEnabled">
                 <bool>false</bool>
                </property>
                <property name="dragDropOverwriteMode">
                 <bool>false</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::NoDragDrop</enum>
                </property>
                <property name="defaultDropAction">
                 <enum>Qt::MoveAction</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>false</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
                <property name="isWrapping" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="itemAlignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="nearestTimer" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="doNotDisturb">
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>24</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>–ù–µ —Ç—É—Ä–±—É–≤–∞—Ç–∏ üò¥</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="regularTimers">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QListWidget" name="timersList">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>JetBrains Mono NL</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="cursor" stdset="0">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="tabKeyNavigation">
                 <bool>true</bool>
                </property>
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="dragEnabled">
                 <bool>true</bool>
                </property>
                <property name="dragDropOverwriteMode">
                 <bool>true</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::DragDrop</enum>
                </property>
                <property name="defaultDropAction">
                 <enum>Qt::MoveAction</enum>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="itemAlignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
